README.md

```diff
@@ -0,0 +1,30 @@
+# schaetzmeisterin
+
+Schaetzmeisterschaft is a fun game to play in the company of your beloved friends. It works just like Texas Hold'em poker. But you will have to guess on a question instead of getting dealt cards. After the showdown, the player with the closest guess gets the pot.
+With schätzmeisterin it is possible, to play schätzmeisteschaft online. 
+
+## Rules
+### Questioning and answering
+- For each hand, one of the players is assigned to be the *dealer*. The dealer must ask a question and provide the correct answer.
+- The other players will then have to submit their guesses.
+
+### Betting
+- The bet-players, that is everybody, except for the dealer, will then take turns in taking bet actions:
+    - Check, fold, raise or call (please refer to the poker-glossary of your choice for explanation of these terms)
+- There is no limit.
+
+### Showdown
+- A question is evaluated, once all players have once submitted any kind of bet, and the remaining players (those who have not folded) all have pushed the same mount of money to the pot.
+- The remaining player with the guess, that is closest, to the correct answer will be rewarded with the pot.
+
+## License
+
+#### Released under MIT License
```

Comment:
There is a better way to add a license on GitHub https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/adding-a-license-to-a-repository

Reply:
thx